autofs-5.0.7 - fix inconsistent use of cache lock in handle_packet_missing_direct()

From: Ian Kent <raven@themaw.net>

All references here except this one refer to the same variable so change the odd
one out for consistency.
---
 daemon/direct.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/daemon/direct.c b/daemon/direct.c
index 228a666..399ad0a 100644
--- a/daemon/direct.c
+++ b/daemon/direct.c
@@ -1392,7 +1392,7 @@ int handle_packet_missing_direct(struct autofs_point *ap, autofs_packet_missing_
 		ops->send_fail(ap->logopt,
 			       ioctlfd, pkt->wait_queue_token, -ENOENT);
 		ops->close(ap->logopt, ioctlfd);
-		cache_unlock(me->mc);
+		cache_unlock(mc);
 		master_source_unlock(ap->entry);
 		master_mutex_unlock();
 		pthread_setcancelstate(state, NULL);
